packages/renderer/src/lib/ui/Typeahead.svelte

```diff
@@ -11,6 +11,7 @@ export let disabled: boolean = false;
 export let initialFocus: boolean = false;
 export let id: string | undefined = undefined;
 export let name: string | undefined = undefined;
+export let isShortName: boolean = false;
```

Comment:
I do not think the TypeaHead component should be responsible of validating the input.

To avoid adding more complexity to the component; the parent should be responsible to define the state (error, okey) of the component. Here is an example https://flowbite-svelte.com/docs/forms/input-field#Validation

Reply:
I confirm here that Typeahead is agnostic as how it's used (so short name is not something valid for the component)